### ISSUE TITLE
encoder updated twice for curl loss

### ISSUE THREAD

**Issue by xy9485** [MishaLaskin/curl#28]
I'm looking into the code and find that in `def update_cpc()`  both `self.encoder_optimizer.step()` and `self.cpc_optimizer.step()` are called. However the parameters of `critic.encoder` are carried by both optimizer. Isn't it true that, in `def update_cpc()`, `critic.encoder` is updated twice using the same gradient?

**Comment by Penelope-zhang** [comment]
@xy9485 do you know why `critic.encoder` is updated twice in `def update_cpc()`?

**Comment by LYK-love** [comment]
Yeah that's true. I don't know the reason of doing this.
